### PR TITLE
Issue #22 Add Gherkin tests for testing CLI functionality

### PR DESCRIPTION
**Pull request by jsliacan**
Signed-off-by: jsliacan <jsliacan@redhat.com>

### CODE REVIEW

File: test/integration/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# CRC integration tests
```
Comment (robin-owen) [inline test/integration/README.md]: Do we want to add this in `test/integration`, or do we want it with the rest of the documentation in `docs`? (I'm working on putting a PR together to start laying the foundations for our docs work.)

By the way, I'd prefer to see all of our documentation in AsciiDoc. If we're fine having this documentation in `test/integration`, then I also have a few comments on the wording and style of this document. Do you mind if I open a PR for your branch to demonstrate if we'll be keeping this in `test/integration`? :)

Comment (jsliacan) [reply test/integration/README.md]: I'm not sure what our preferences are about where to have it! Maybe @tsedmik knows? 

Go ahead with PRs as you need/want :) Thanks!

Comment (tsedmik) [reply test/integration/README.md]: Let's keep all docs together :-).

**Comment by gbraad** [comment]
Can we update this PR (and rebase against the current master)

**Comment by gbraad** [comment]
Closing this in favour of #95

**Pull request closed without merging**